certified-connectors/Plumsail Documents/readme.md

```diff
@@ -0,0 +1,18 @@
+## Plumsail Documents Connector
+Plumsail Documents provides a powerful and very extensive REST API.  Using this API, you can generate documents in Microsoft Flow, Azure Logic Apps or PowerApps. 
+
+## Pre-requisites
+You will need the following to proceed:
+* A Microsoft Power Apps or Power Automate plan with custom connector feature
+* A valid [Plumsail Account](https://account.plumsail.com)
+* The Power platform CLI tools
+
+### Deploying the sample
+Run the following commands and follow the prompts:
+
+```paconn
+paconn create --api-def apiDefinition.swagger.json --api-prop apiProperties.json --secret <client_secret>
```

Comment:
This specific command is for connectors that have OAuth has their auth method. It looks like yours uses API key, so this should instead be changed to:

`paconn create --api-prop [Path to apiProperties.json] --api-def [Path to apiDefinition.swagger.json] --icon [Path to icon.png]`

Reply:
Thanks for updating!